templates/workers.yaml.j2

```diff
@@ -6,7 +12,7 @@ metadata:
     app: "celery"
   namespace: "{{ k8s_namespace }}"
 type: Opaque
-stringData: {{ k8s_worker_container["environment"] | to_yaml }}
+stringData: {{ k8s_worker_container["environment"] | to_json }}
```

Comment:
Does this need to be base64 encoded? (Still haven't quite figured out these Secret things.)

Reply:
`stringData` doesn't, `data` does, afaik: https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret-manually